tests/cargo-kani/mir-linker/src/lib.rs

```diff
@@ -5,6 +5,7 @@
 use semver::{BuildMetadata, Prerelease, Version};
 
 #[kani::proof]
+#[kani::unwind(2)]
```

Comment:
Is it worth investigating why this is newly necessary?

Reply:
I meant to file an issue to track this. Thanks for the reminder.

Reply:
https://github.com/model-checking/kani/issues/1978